src/parser/tokenizer/yaml/index.js

```diff
@@ -0,0 +1,33 @@
+const yaml = require('js-yaml');
+const { flatten, uniq } = require('lodash');
+
+const toArray = (obj, arr) => {
+  if (obj) {
+    Object.keys(obj).forEach((key) => {
+      const value = obj[key];
+      if (value !== null && typeof value === 'object') {
+        toArray(value, arr)
+      } else if (value) {
+        arr.push(value.toString());
+        }
+    });
+  }
+  return arr;
+};
+
+module.exports = (content, config) => {
+  const result = [];
+  let object;
+
+  if (config) {};
```

Comment:
Is this needed?

Reply:
originally thought i'd need the module to accept the config argument so then would need to use the variable, removed now